### PR TITLE
feat: add --from flag at get-oracle-key

### LINKED ISSUE #70: Add get-oracle-key CLI at OracleUpgrade store
Currently the `get-oracle-key` CLI get the `EncryptedOraclePrivKey` from only `OracleRegistration`.
But in oracle upgrade we can't use this CLI in new oracle (Because the `nodePrivKey` is different). Therefore, if storing `OraclePrivKey` fails for some reason during upgrade, there is no way to obtain the key again.
So, It is necessary to improve `get-oracle-key` for covering upgrade scenario.

### PR DESCRIPTION
**Pull request by audtlr24**
closes #70 .

To solve the issue, I added a flag `--from` to `get-oracle-key` CLI.
I also thought about how to automatically import the oracle key when upgraded, but adding a flag seemed the simplest to cover all cases of key loss. If there is a better way, please comment.

When I tested it in local environment, I confirmed that it works well.

### CODE REVIEW

File: cmd/oracled/cmd/get_oracle_key.go [hunk 1]
```diff
@@ -28,18 +29,40 @@ func getOracleKeyCmd() *cobra.Command {
...
+
+			from, err := cmd.Flags().GetString(flags.FlagFromOracleRegistrationOrUpgrade)
 			if err != nil {
-				return fmt.Errorf("failed to get oracle registration: %w", err)
+				return err
 			}
 
-			if len(oracleRegistration.EncryptedOraclePrivKey) == 0 {
-				return fmt.Errorf("the encrypted oracle private key has not set yet. please try again later")
+			if from == "registration" {
```
Comment (youngjoon-lee) [inline cmd/oracled/cmd/get_oracle_key.go:38]: It'd be safe to define these magic strings as const. Sorry that Go doesn't have enum.
```go
const (
  fromRegistration = "registration"
  fromUpgrade = "upgrade" 
)
```

Or, you can emulate the enum like other programming langs.
```go
type oracleKeyFetchMode string

const (
  fromRegistration oracleKeyFetchMode = "registration"
  fromUpgrade oracleKeyFetchMode = "upgrade" 
)

func parseOracleKeyFetchMode(str string) (oracleKeyFetchMode, error) {
  switch str {
  case: fromRegistration
     return fromRegistration
  ...
  default:
      return fmt.Errorf("invalid...")
  }
}
```
(I didn't check the syntax of these codes).

**Review by youngjoon-lee — APPROVED** [review]
lgtm

File: cmd/oracled/cmd/get_oracle_key.go (hunk 1 above)
Comment (audtlr24) [reply cmd/oracled/cmd/get_oracle_key.go]: Thanks. As you commented, I changed the magic string to const.

File: cmd/oracled/cmd/get_oracle_key.go [hunk 2]
```diff
@@ -28,18 +34,41 @@ func getOracleKeyCmd() *cobra.Command {
...
+				}
+				return key.RetrieveAndStoreOraclePrivKey(ctx, svc, oracleUpgrade.EncryptedOraclePrivKey)
+
+			default:
+				return fmt.Errorf("invalid --from flag input. please put \"registration\" or \"upgrade\"")
+			}
 		},
 	}
 
+	cmd.Flags().String(flags.FlagFromOracleRegistrationOrUpgrade, "upgrade", "where to get the key from")
```
Comment (gyuguen) [inline cmd/oracled/cmd/get_oracle_key.go:71]: Is `upgrade` as the default value intended?
Then I think you can change like below:
```suggestion
	cmd.Flags().String(flags.FlagFromOracleRegistrationOrUpgrade, fromUpgrade, "where to get the key from")
```

**Review by gyuguen — APPROVED** [review]
Thanks. 
I left a comment.

File: cmd/oracled/cmd/get_oracle_key.go (hunk 2 above)
Comment (audtlr24) [reply cmd/oracled/cmd/get_oracle_key.go]: I missed it. Thank you.

**Review by inchori — APPROVED** [review]
lgtm

**Pull request merged**